docs/developers/pf-configuration.md

```diff
@@ -0,0 +1,147 @@
+
+* Common `lookback` and `priceFeedDecimals` parameters.
+* `balancerAddress`: Address of the Balancer pool the price feed is monitoring.
+* `balancerTokenOut`: Base token address (it should be part of `balancerAddress` pool). This price feed will return the price of base token expressed in units of quote token.
+* `balancerTokenIn`: Quote token address (it should be part of `balancerAddress` pool).
+* `twapLength`: Duration in seconds of the time weighted average computation applied by the price feed. When set to 0 this would return spot price for the requested timestamp.
+* `poolDecimals` (optional): Represents the number of decimals that Balancer pool prices are returned in. If not provided it defaults to 18 that is appropriate when both base and quote tokens have the same scaling decimals. If base token has more decimals than quote tokens then their difference should be deducted from 18 in the `poolDecimals`, while for base token having less decimals than quote token this difference should be added to 18. As an illustration, YD-ETH-SEPT (base) has 18 decimals and USDC (quote) has 6 decimals, so deducting their difference of 12 from 18 should leave `poolDecimals: 6`.
+
+### Liquidity Provider
+
+Select [Liquidity Provider](https://github.com/UMAprotocol/protocol/blob/master/packages/financial-templates-lib/src/price-feed/LPPriceFeed.js) price feed by setting `type: "lp"` in the configuration to fetch pricing data of pool tokens expressed in underlying tokens. Note that in order to calculate full LP token value one has to combine price feeds from all pool reserve tokens. Supported configuration parameters are listed below:
+
+* Common `minTimeBetweenUpdates` and `priceFeedDecimals` parameters.
+* `poolAddress`: Address of the LP pool to monitor.
+* `tokenAddress`: Address of the per-share token balance that should be tracked within the pool.
+
+### TraderMade
+
+Select [TraderMade](https://github.com/UMAprotocol/protocol/blob/master/packages/financial-templates-lib/src/price-feed/TraderMadePriceFeed.js) price feed by setting `type: "tradermade"` in the configuration to fetch forex pricing data from TraderMade. Supported configuration parameters are listed below:
+
+* Common `minTimeBetweenUpdates` and `priceFeedDecimals` parameters.
+* `minuteLookback` (optional if `hourlyLookback` is provided): Number of seconds how far in the past the historical minute interval prices will be fetched. Maximum allowed value is 172800 seconds (2 days).
+* `hourlyLookback` (optional): Number of seconds how far in the past the historical hourly prices will be fetched. Hourly historical prices can be used as a fallback to the minute time-series if no minute data is available (e.g. on weekends). Maximum allowed value is 5184000 seconds (2 months).
+* `tradermadeApiKey`: This is an application specific parameter, where the user should provide its TraderMade account API key.
```

Comment:
Think it is worth being more explicit that this should not be included in `DefaultPriceFeedConfigs.js`? Seems obvious, and I know you note that it is an application specific parameter, but thoughts on calling it out again?

Reply:
Added warning both to CW and TM